accounts.go

```diff
@@ -58,6 +58,22 @@ func (am accountManager) CreateAccount(acctName string, email string, db *gorm.D
 		return nil, err
 	}
 
+	var instance serviceInstance
+	err = db.First(&instance, "instance_id = ?", "available").Error
+
+	if err == nil {
```

Comment:
More Go-y to do this the other way around: check for `err` and return it if not `nil`.